source/whats-new.txt

```diff
@@ -49,105 +49,105 @@ The following changes affect future versions of the driver:
 What's New in 4.11
 ------------------
 
-.. important:: Deprecation Notice
+This section includes the following information:
+
+- :ref:`java-deprecations-4.11`
+- :ref:`java-new-features-4.11`
+
+.. _java-deprecations-4.11:
+
+Deprecations in 4.11
+~~~~~~~~~~~~~~~~~~~~
+
+.. warning:: Deprecations in this version
 
-   - The ``getStats()`` and ``isCapped()`` instance methods of the
-     ``DBCollection`` class are deprecated. The corresponding server
-     commands are deprecated in MongoDB v6.2 and later. Use the ``$collStats`` aggregation
-     pipeline stage to retrieve the information provided by these methods
-     instead. You can run the aggregation as shown in the following code example:
+   This version of the driver deprecates classes, methods, or both.
+   To avoid breaking changes in a future version of the driver,
+   replace any application code that depends on the deprecated code.
 
-     .. code-block:: java
+The 4.11 driver release deprecates the following items:
 
-        collection.aggregate(Arrays.asList(
```

Comment:
It's a little weird that the replacement for a method in DBCollection is given here using MongoCollection (it doesn't actually say that, but it is).  Let's show an example with DBCollectoin.aggregate instead.